framework/include/kernels/ADKernel.h

```diff
@@ -52,7 +53,9 @@
   using ADKernelTempl<type, compute_stage>::accumulateTaggedLocalMatrix;                           \
   using ADKernelTempl<type, compute_stage>::variable;                                              \
   using ADKernelTempl<type, compute_stage>::paramError;                                            \
-  using ADKernelTempl<type, compute_stage>::isParamValid
+  using ADKernelTempl<type, compute_stage>::isParamValid;                                          \
```

Comment:
This may not be necessary any more since its in usingMooseObjectMembers...

Reply:
Right, this can be removed